### ISSUE TITLE
Zalew Włocławski - river shapefile and lake shapefile misaligned

### ISSUE THREAD

**Issue by matkoniecz** [nvkelso/natural-earth-vector#486]
This issue concerns mismatch between datasets concerning single waterbody - https://www.openstreetmap.org/relation/2329477/history#map=11/52.5968/19.4362

in 10m datasets rivers is outside its own reservoir what makes no sense

![screen01](https://user-images.githubusercontent.com/899988/110099617-ca2bb200-7da1-11eb-9af4-cd91529c974f.png)

I would be happy to try contribute a fix, using Geoportal.gov.pl imagery available without limits on its use.

Should I open https://github.com/nvkelso/natural-earth-vector/tree/master/10m_physical in QGIS, save it and submit PR?

(BTW, contributing.md file where info about how typical contribution proceeds seems missing)

**Comment by matkoniecz** [comment]
Should I open https://github.com/nvkelso/natural-earth-vector/tree/master/10m_physical in QGIS, edit it, save it and submit PR?

(I would need to figure out how to get Geoportal.gov.pl aerial imagery WMS/TMS into QGIS)

Or is changing geometries done using other method?

Or maybe it is considered as acceptable and changes would be rejected?

(I somehow missed "edit" part previously)

**Comment by nvkelso** [comment]
Yes, you can edit the SHP and submit in PR. Agree this lake needs some adjustment. Please limit PR to just these features and maybe the other lake name change so it’s easier to review. Thanks!

**Comment by nvkelso** [comment]
As @matkoniecz points out in the PR https://github.com/nvkelso/natural-earth-vector/pull/491 - it seems that supplemental lakes dataset at least in Poland and all of Europe are generally moved toward North by some vector (seems about 800m).

![screen04](https://user-images.githubusercontent.com/899988/110325214-58f53480-8017-11eb-9e19-a07a91720fbe.png)

(above) Łebsko Lake as a random example

Generally it seems like all the features in the layer should move south that amount... but then what about the supplemental Europe rivers? It seems like they should move south too... but some care was taken before to connect those to the basic river network. So this could turn out to be a significant chunk of work. Perhaps we tackle this by country?

Let's see how much @elliotap and @MrGawsome can get done for v5.1 in a sprint to move them all around. Whatever is left let's peel off as separate per country issues for the rest.

**Comment by nvkelso** [comment]
Tom is planning fixes for all hydro as part of the 6.x series.

**Comment by matkoniecz** [comment]
Would you potentially accept PR fixing all such misaligned objects across Poland?

**Comment by nvkelso** [comment]
@matkoniecz Sorry to miss your message earlier because of the holidays. Yes, now that the v5 series is released we would accept a PR fixing misaligned hydro (default rivers and lakes, Europe supplement for same). 

An update on Tom's work: he's gotten the new coastline in alpha state globally is working with a few folks to add supplemental hydro in South America, Africa, and Asia to match the details we have in North America, Europe, and Australia. There is a Slack channel we can invite you to as that may be easier to coordinate your work as it comes in?
